docs/book/src/cronjob-tutorial/testdata/project/test/utils/utils.go

```diff
@@ -167,9 +167,9 @@ func IsCertManagerCRDsInstalled() bool {
 
 // LoadImageToKindClusterWithName loads a local docker image to the kind cluster
 func LoadImageToKindClusterWithName(name string) error {
-	cluster := "kind"
-	if v, ok := os.LookupEnv("KIND_CLUSTER"); ok {
-		cluster = v
+	cluster, ok := os.LookupEnv("KIND_CLUSTER")
+	if !ok {
+		return fmt.Errorf("KIND_CLUSTER is not set")
```

Comment:
Do we need this change?

Reply:
It would have worked with the old code, but since we no longer have a default `kind` cluster, it’s not necessary anymore. However, if the environment variable isn’t set, the image deployment will fail. I’d feel more comfortable if we explicitly notify the user when it's missing.

Reply:
I will need some time to test this one.
If I just have kind running ( any cluster name ) and run the e2e tests, will that work fine? 
If so, I am OK with if not I think we should allow it.

Reply:
I get your point. I think you will start the e2e test right out of your IDE. I will have to think about that and implement it. Currently that wouldn't work out of the box. You would have to provide the ENV var in the IDE launch config. I will provide a way, where it defaults to `kind`.

Reply:
> Currently that wouldn't work out of the box

I need to check it, but I think what you are saying today does not work if we have any cluster name created right?

Reply:
So, I think we should revert the change in the utils for now.
If needed, we can revisit and improve or refactor it in a separate PR since it feels a bit out of scope for this one.

That way, we can move forward with the changes in the Makefile without blocking.
WDYT?